config/jest/fileTransform.js

```diff
@@ -6,5 +6,5 @@ const path = require("path");
 module.exports = {
   process(src, filename) {
     return `module.exports = ${JSON.stringify(path.basename(filename))};`;
-  }
```

Comment:
Some of files is changed to using comma seperator on last file. Is there any difference?
Or just rules of lint?

Reply:
I changed `package.json` file's `lint` command like [this](https://github.com/project-dobby/dobby/pull/11/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23).
So some of files were newly included to eslint target.